pages/_app.tsx

```diff
@@ -5,9 +5,9 @@ import '../css/tailwind.css';
 
 const MyApp: FC<AppProps> = ({ Component, pageProps }) => {
   return (
-    <main className="min-h-screen bg-gray-400">
+    <div className="min-h-screen bg-gray-400">
```

Comment:
Why?

Reply:
I updated the `index.tsx` to have the `main`-tag, which will contain the actual page content.